src/librustc_llvm/lib.rs

```diff
@@ -432,3 +432,4 @@ impl Drop for OperandBundleDef {
 mod llvmdeps {
     include! { env!("CFG_LLVM_LINKAGE_FILE") }
 }
+#[link(name = "ffi")] extern {}
```

Comment:
TODO remove. (Sometimes I forget to not commit this -- workaround for rust-lang/rust#34486)